settings/tarantool_test.go

```diff
@@ -481,7 +481,14 @@ func TestSQLReverseUnorderedSelectsSetting(t *testing.T) {
 	require.Equal(t, []interface{}{[]interface{}{"sql_reverse_unordered_selects", false}}, resp.Data)
 
 	// Select multiple records.
-	resp, err = conn.Execute("SELECT * FROM data;", []interface{}{})
+	query := "SELECT * FROM seqscan data;"
+	if isSeqScanOld, err := test_helpers.IsTarantoolVersionLess(3, 0, 0); err != nil {
+		t.Fatal("Could not check the Tarantool version")
+	} else if isSeqScanOld {
```

Comment:
Btw, Seqscan was introduced in 2.11.0-rc1 (but it's not required by default), so you may use this condition too.

Reply:
I think it is a good idea to test the default, which is 'old' for 2.11 and 'new` (with seqscan key word) in 3.0 and up.